types/knex/index.d.ts

```diff
@@ -439,8 +440,8 @@ declare namespace Knex {
         primary(columnNames: string[]): TableBuilder;
         index(columnNames: (string | Raw)[], indexName?: string, indexType?: string): TableBuilder;
         unique(columnNames: (string | Raw)[], indexName?: string): TableBuilder;
-        foreign(column: string): ForeignConstraintBuilder;
-        foreign(columns: string[]): MultikeyForeignConstraintBuilder;
+        foreign(column: string, foreignKeyName?: string): ForeignConstraintBuilder;
+        foreign(columns: string[], foreignKeyName?: string): ForeignConstraintBuilder;
```

Comment:
Was the `MultikeyForeignConstraintBuilder` a bug or is the return type a `ForeignConstraintBuilder` when `foreignKeyName` is provided and a `MultikeyForeignConstraintBuilder` when omitted?

Reply:
That would be a mistake on my part.  I will correct.